care/emr/api/viewsets/activity_definition.py

```
@@ -126,12 +126,20 @@ def validate_health_care_service(self, instance):
             raise ValidationError("Healthcare Service must be from the same facility")
 
+    def validate_data(self, instance, model_obj=None):
+        queryset = ActivityDefinition.objects.filter(slug__iexact=instance.slug)
+        if model_obj:
+            queryset = queryset.filter(facility=model_obj.facility).exclude(
+                id=model_obj.id
+            )
+        else:
+            queryset = queryset.filter(facility=self.get_facility_obj())
+        if queryset.exists():
+            raise ValidationError("Slug already exists in this facility.")
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)